src/pages/index.jsx

```diff
@@ -45,7 +45,7 @@ export default function Home() {
           <div className="container">
             <div className="row">
               <div className="col col--6">
-                <div className="alternate-sponsors">
+                <div className="alternate-sponsors" style={{ display: '100%' }}>
```

Comment:
Is it a valid CSS?
Shouldn't `display` accept `block`, `inline-block`, etc.
I never see `display: 100%`

Reply:
I don’t know why it became display lol, it ahould be height 100%